test/new-e2e/go.mod

```diff
@@ -12,6 +12,8 @@ replace (
 	github.com/DataDog/datadog-agent/pkg/util/testutil => ../../pkg/util/testutil
 	github.com/DataDog/datadog-agent/pkg/version => ../../pkg/version
 	github.com/DataDog/datadog-agent/test/fakeintake => ../fakeintake
+	github.com/DataDog/test-infra-definitions => ../../../test-infra-definitions
```

Comment:
<!-- Warning highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes and they are blockers. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question. -->
**🔨 warning**
```suggestion
```
we cannot replace out of the repo

Reply:
Yes I was not supposed to commit this... Sorry

Reply:
No worries !